201-front-door-standard-premium-rate-limit/main.bicep

```diff
@@ -0,0 +1,220 @@
+@description('The name of the Front Door endpoint to create. This must be globally unique.')
+param endpointName string = 'afd-${uniqueString(resourceGroup().id)}'
+
+@description('The host name that should be used when connecting from Front Door to the origin.')
+param originHostName string
+
+@description('The name of the SKU to use when creating the Front Door profile.')
+@allowed([
+  'Standard_AzureFrontDoor'
+  'Premium_AzureFrontDoor'
+])
+param frontDoorSkuName string = 'Standard_AzureFrontDoor'
+
+@description('The mode that the WAF should be deployed using. In \'Prevention\' mode, the WAF will block requests it detects as malicious. In \'Detection\' mode, the WAF will not block requests and will simply log the request.')

```

Comment:
ok, I suspect the escaping quotes was probably a pattern I just missed before... flagging it here in case you want to update them.

Reply:
Given these are rendered using Markdown (as far as I know at least?) in the portal UI and potentially other tooling, I felt that using the backticks was the right thing to do for literal values. If you'd like me to replace them with double quotes I can though.

Reply:
Portal will handle either - so I'm optimizing for making the bicep source look pretty.  Right now the bicep needs escaping and the json is pretty.  I'm thinking we should flip it.

Reply:
Updated.